packages/jw-player/src/plugins/telemetry-8.20.0.js

```diff
@@ -0,0 +1,192 @@
+/* eslint-disable */
+
+// Rudder Stack Telemetry Plugin v8.1.0 Wed May 11 2022 02:20:30 GMT+1000 (Australian Eastern Standard Time)
```

Comment:
Why we have this file?

Can't we use official sdk?
Ref: https://github.com/rudderlabs/rudder-sdk-js

Sample usage: https://github.com/rudderlabs/rudder-analytics-next/blob/4c3c6bb2d50c5c591c855d414832cb7c7fc1d390/sample-next-app-npm/pages/rudderInitialize.js#L2

Reply:
This was submitted by one of our outside developers. We don't have much control right now what goes into the file.

Reply:
> This was submitted by one of our outside developers. We don't have much control right now what goes into the file.
What you mean we don't have control? 
Its looks like a compiled / transpiled file, also no source of origin, where it was inspired from, based on which code would help too.

Reply:
We just go access to the plugin source: https://github.com/conversionxl/cxl-jwplayer-plugins

In order for us to use/test it, it needs to be hosted:

https://github.com/conversionxl/aybolit/blob/1124d4b2f897320946e1dd26b1e0ba068bf411cf/packages/jw-player/src/components/jw-player/index.js#L21